lte/cloud/go/services/subscriberdb/obsidian/handlers/handlers.go

```diff
@@ -648,11 +750,16 @@ func deleteSubscriber(networkID, key string) error {
 	return nil
 }
 
-func loadAllStatesBySID(networkID string) (map[string]state_types.StatesByID, error) {
-	states, err := state.SearchStates(networkID, subscriberStateTypes, nil, nil, serdes.State)
+func loadAllStatesBySIDForIMSIs(networkID string, imsis []string) (map[string]state_types.StatesByID, error) {
```

Comment:
`loadAllStatesBySIDForIMSIs` -> `loadStatesByIMSI`, and in doc comment indicate that its for the passed imsis